Frontend/CO.CDP.OrganisationApp/appsettings.json

```diff
@@ -55,5 +63,7 @@
   "CharityCommission": {
     "Url": "",
     "SubscriptionKey": ""
-  }
+  },
+  "OneLoginLogoutNotificationUrls": [
```

Comment:
Looks like result of merge conflict, `OneLoginLogoutNotificationUrls `is not required to be added